common/azure_auth_test.go

```diff
@@ -382,29 +392,6 @@ func TestAzureEnvironment(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
-func TestInvalidAzureEnvironment(t *testing.T) {
-	aa := AzureAuth{}
-
-	aa.Environment = "xyzdummy"
-	_, envErr := aa.getAzureEnvironment()
-	assert.NotNil(t, envErr)
-
-	mockFunc := func(resource string) (autorest.Authorizer, error) {
-		return nil, nil
-	}
-
-	ctx := context.Background()
-	_, err := aa.simpleAADRequestVisitor(ctx, mockFunc)
-
-	assert.Equal(t, envErr, err)
-
-	_, err = aa.acquirePAT(ctx, mockFunc)
-	assert.Equal(t, envErr, err)
-
-	_, err = aa.getClientSecretAuthorizer("")
-	assert.Equal(t, envErr, err)
-}
-
```

Comment:
these functions are not calling getAzureEnvironment anymore making this test kind of pointless to have.